<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>saldoWindow</class>
 <widget class="QWidget" name="saldoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Saldo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/icons/talvi.jpg);</string>
  </property>
  <widget class="QLabel" name="labelSaldo">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>70</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="suljeButton">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>250</y>
     <width>150</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SULJE</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAsiakas">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>10</y>
     <width>461</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLCDNumber" name="timer">
   <property name="geometry">
    <rect>
     <x>1730</x>
     <y>930</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>631</width>
     <height>221</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
